### ISSUE TITLE
shellenv missing - through packagecontrol.io

### ISSUE THREAD

**Issue by ucirello** [alvarolm/GoGuru#12]
```
reloading plugin GoGuru.goGuru
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 120, in reload_plugin
    m.plugin_loaded()
  File "goGuru in /Users/uldericofilho/Library/Application Support/Sublime Text 3/Installed Packages/GoGuru.sublime-package", line 57, in plugin_loaded
  File "goGuru in /Users/uldericofilho/Library/Application Support/Sublime Text 3/Installed Packages/GoGuru.sublime-package", line 43, in load_shellenv
ImportError: No module named 'shellenv'
```

![image](https://cloud.githubusercontent.com/assets/138827/15124812/1b79771e-1600-11e6-949c-efb53b06e342.png)

**Comment by alvarolm** [comment]
fixed ! thanks for reporting :)
https://github.com/alvarolm/GoGuru/issues/11

**Comment by alvarolm** [comment]
it should be fixed as version 0.1.4, the package control uses 1 hour intervals to pull updates from the git repo, so we just have to wait, I have tested anyway on my machine and seems to be fixed.

**Comment by alvarolm** [comment]
please feel free to close this one as soon its working for you.
